src/ErrorHandler.php

```diff
@@ -69,7 +69,7 @@ public function handle(
         $renderer ??= $this->defaultRenderer;
 
         try {
-            $this->logger->error(PlainTextRenderer::throwableToString($t), ['throwable' => $t]);
+            $this->logger->error(PlainTextRenderer::throwableToString($t));
```

Comment:
```suggestion
            $this->logger->error($t->getMessage(), ['throwable' => $t]);
```
or
```suggestion
            $this->logger->error($t);
```

I'd leave it as the suggested